src/Http/Routing/src/EndpointGroupNameAttribute.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.AspNetCore.Http;
+ 
+ namespace Microsoft.AspNetCore.Routing
+ {
+    /// <summary>
+    /// Specifies the endpoint group name in <see cref="Microsoft.AspNetCore.Http.Endpoint.Metadata"/>.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Delegate | AttributeTargets.Class, Inherited = false, AllowMultiple = false)]
+    public sealed class EndpointGroupNameAttribute : Attribute, IEndpointGroupNameMetadata
```

Comment:
Do we need to change MVC to consume this?

Reply:
As in ensure it adds it to the `ApiExplorer`? Probably.